### PR TITLE
Aqara Radiator Thermostat E1 SRTS-A01 lumi.airrtc.agl001

### PR DESCRIPTION
**Pull request by kirovilya**
Basic functionality.
No battery. There is no schedule. No external sensor

### CODE REVIEW

File: devices/xiaomi.js [hunk 1]
```diff
@@ -2164,4 +2252,20 @@ module.exports = [
...
+        fromZigbee: [fzLocal.aqara_trv, fz.thermostat],
+        toZigbee: [tzLocal.aqara_trv, tz.thermostat_occupied_heating_setpoint],
+        exposes: [e.switch().setAccess('state', ea.STATE_SET),
+            exposes.climate().withSetpoint('occupied_heating_setpoint', 5, 30, 0.5)
+                .withLocalTemperature(ea.STATE).withPreset(['manual', 'away', 'auto'], ea.STATE_SET),
+            e.child_lock(), e.window_detection(), e.valve_detection(),
+            e.away_preset_temperature(),
+        ],
+        configure: async (device, coordinatorEndpoint, logger) => {
+        },
```
Comment (Koenkk) [inline devices/xiaomi.js:2269]: Was this left empty on purpose? If not please remove.

Comment (kirovilya) [reply devices/xiaomi.js:2269]: can be removed for now :)

Comment (kirovilya) [reply devices/xiaomi.js]: done

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**